### ISSUE TITLE
Input Field is not being announced

### ISSUE THREAD

**Issue by ThaMoreira** [FreedomScientific/VFO-standards-support#674]
Summary
If you have a input JAWS will not inform about that it is a field that you can edit or to type and text

Example:

Go to the URL https://sapui5.hana.ondemand.com/sdk/#/api/sap.m.OverflowToolbar%23methods/Summary
Try to read the Filter Input Field
JAWS will not inform you that you can type and text on this field

Expected result
JAWS will inform the user that the input is available by reading out something to the effect of "... Input field Type and text..."

Actual result
JAWS does not indicate the input field and does not read anything

<img width="453" alt="Untitled" src="https://user-images.githubusercontent.com/36004615/199792068-1aa70424-ab32-4da4-954f-a0d2988d5c15.png">

**Comment by JAWS-test** [comment]
The problem does not occur with JAWS 2021, but does occur with JAWS 2022 and Chrome. When reading linearly, JAWS switches to form mode but does not output that it is in an input field. With TAB operation, the output is correct
